src/components/TableSchema/index.jsx

```diff
@@ -11,7 +11,10 @@ import "./TableSchema.css";
 
 const TableSchema = (props) => {
   const [schema, setSchema] = useState(props.schema);
-  const [unfilledRichTypes, setUnfilledRichTypes] = useState(props.schema.fields.length - 1);
+  //add 2 to the length of the schema to take account of description and tittle field
+  const [unfilledRichTypes, setUnfilledRichTypes] = useState((props.schema.fields.length+2) - 1);
```

Comment:
@steveoni a suggestion. Wouldn't it be cleaner to track all fields with a single state object?

Reply:
A single state object is actually used to track the two other important field as suggested by the issue.

the idea of adding two to the unfilled rich type is base on the fact that the initial design depends on the rich field type to move to the next screen, so to prevent checking for lot of conditions I use this.

so this implementation track the description and title state but assume it's part of the richfield type

Reply:
the Description and title are single field or for each column?